### PR TITLE
Update chromium from 665556 to 667462

### PR DESCRIPTION
**Pull request by ran-dall**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

**Comment by ran-dall** [comment]
Supersedes 👆 https://github.com/Homebrew/homebrew-cask/pull/64496

File: Casks/chromium.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 cask 'chromium' do
-  version '665556'
-  sha256 '093c466afd98f18b24608737ee67f9afa4fd9cebe052782181b3e49c310175ee'
+  version '667462'
```
Comment (vitorgalvao) [inline Casks/chromium.rb]: Already at `667463`.

Comment (ran-dall) [reply Casks/chromium.rb]: So should we maybe would this be a good candidate to tag as `:latest`?

**Comment by ran-dall** [comment]
Superseded 👆https://github.com/Homebrew/homebrew-cask/pull/64575

**Pull request closed without merging**